### PR TITLE
federate node kube_node_info to metering prometheus

### PR DESCRIPTION
**Pull request by mate4st**
**What this PR does / why we need it**:

Adds a new metric to metering Prometheus 

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**

/kind bug
/kind feature

**Special notes for your reviewer**:

Release not in the corresponding metering PR: 
https://github.com/kubermatic/metering/pull/144

```release-note
NONE
```

**Documentation**:

```documentation
NONE 
```

### CODE REVIEW

File: pkg/ee/metering/prometheus/reconcile.go [hunk 1]
```diff
@@ -71,7 +72,7 @@ func ReconcilePrometheus(ctx context.Context, client ctrlruntimeclient.Client, s
 
 	if err := reconciling.ReconcileStatefulSets(ctx, []reconciling.NamedStatefulSetReconcilerFactory{
 		prometheusStatefulSet(getRegistry, seed),
-	}, seed.Namespace, client, common.VolumeRevisionLabelsModifierFactory(ctx, client), seedOwner); err != nil {
+	}, seed.Namespace, client, common.VolumeRevisionLabelsModifierFactory(ctx, client), seedOwner, volumeLabelModifier); err != nil {
```
Comment (xrstf) [inline pkg/ee/metering/prometheus/reconcile.go]: This is redundant, no?

Comment (mate4st) [reply pkg/ee/metering/prometheus/reconcile.go]: yep wtf :D

**Review by xrstf — APPROVED** [review]
/approve

**Comment by embik** [comment]
@WeirdMachine will this need a backport?

**Comment by WeirdMachine** [comment]
> @WeirdMachine will this need a backport?

Good question. I think it makes sense to backport it up to v2.20 so we can patch metering there as well.

**Pull request merged**